### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by TheNeg0t1ator**
Final merge of neopixel to main

### CODE REVIEW

File: .github/workflows/main.yaml
Comment (SeppeBudenaers) [inline .github/workflows/main.yaml:1]: maybe not change this file

**Review by SeppeBudenaers — APPROVED** [review]

**Review by SeppeBudenaers — APPROVED** [review]
Looks good to me

**Comment by SamyWarnants** [comment]
Looks good to me!

**Review by SamyWarnants — APPROVED** [review]
It is quite a lot but I don't see anything wrong so looks good to me!

**Comment by RunningZeus5334** [comment]
:+1:

**Pull request merged**